androidgen/src/main/java/com/azure/autorest/android/template/AndroidServiceAsyncClientTemplate.java

```diff
@@ -24,4 +24,9 @@ protected void addServiceClientAnnotationImports(java.util.Set<String> imports)
     @Override
     protected void addGeneratedAnnotation(JavaContext classBlock) {
     }
+
+    @Override
+    protected boolean isPartialUpdateSupported() {
```

Comment:
Given Android generates targeting Java as well, do we want to just enable this? At the end of the day it's just a set of XML comments. Plus, given the limited usage of the Android generator it would simplify code in Javagen.

Comment applies to the other spots in Androidgen as well.

Reply:
From what I remember, Android turns it off because their azure-core does not have `Generated` annotation.

Not sure if it's changed though.